tests/visual/notification.html

```diff
@@ -92,13 +107,288 @@
             <div class="k-animation-container" style="overflow: visible; display: block; position: static">
                 <div class="k-widget k-popup k-notification k-notification-warning k-state-border-up" style="display: block; ">
                     <div class="k-notification-wrap">
-                        <span class="k-icon k-i-error" title="error"></span>
+                        <span class="k-icon k-i-warning" title="warning"></span>
                         Warning
                         <span class="k-icon k-i-close" title="Hide"></span>
                     </div>
                 </div>
             </div>
-        </div>
+        </section>
+
+        <!-- angular notifications -->
+        <section class="dark">
+            <div class="k-animation-container k-animation-container-fixed k-animation-container-shown">
+                <div class="k-popup k-popup-transparent">
+                    <div class="k-widget k-notification k-notification-info">
+                        <div class="k-notification-wrap">
+                            <span class="k-icon k-i-info"></span>
+                            <div class="k-notification-content">
+                                Hi! I am SUCCESS notification
```

Comment:
I'd suggest writing the framework flavor in the notification-content. This allows the screenshot to be much easier to understand, e.g. "Success notification in Angular" and "Closable info notification in jQuery"

Reply:
Agreed on this one